frontend/src/pages/About.js

```diff
@@ -1,9 +1,28 @@
 import React from 'react';
+import { iconAdd, iconCart, iconCheck } from '../components/ui-kit/icons';
+import Button from '../components/ui-kit/components/Button/Button';
```

Comment:
Since we have an index.js file, you don't need to go directly to the file, just to the folder